### PR TITLE
Added User Account 

### PR DESCRIPTION
**Pull request by tushartiwari7**
User account will have following sub pages - 
- Profile Page
  -  Can see Firstname, Lastname and mail address.
  - Feat: Can edit your name and mail address.
- Address Page for Address Management
  - Feat:  Add a new address
  - Feat: Can see a list of addresses.
  - Can Edit/Remove address from address list.
- Settings Page
  - Feat: Can reset all personalized user data  like address,
  - Feat: Can Delete User from Db.
 - Checkout Page
   -  Choose from the addresses where you want the item to be delivered.

### CODE REVIEW

File: src/services/address.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
+import toast from "react-hot-toast";
+
+export const addAddress = async (address) => {
+  const token = localStorage.getItem("token").slice(1, -1);
+  try {
+    const { data, status } = await axios({
+      method: "POST",
+      url: `/api/user/addresses`,
```
Comment (ashishnnnnn) [inline src/services/address.js:9]: Liked that you managed the mock-back-end for address also..

File: src/pages/MyAccount/MyAccount.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      routeName: "Profile",
+    },
+    {
+      route: "addresses",
+      routeName: "Manage Addresses",
+    },
+    // {
+    //   route: "change-password",
+    //   routeName: "Change Password",
+    // },
```
Comment (ashishnnnnn) [inline src/pages/MyAccount/MyAccount.jsx:27]: Can remove the commented code..

**Review by ashishnnnnn — APPROVED** [review]
Overall code looks good :+1:

File: src/pages/MyAccount/MyAccount.jsx (hunk 2 above)
Comment (tushartiwari7) [reply src/pages/MyAccount/MyAccount.jsx:27]: okay

**Review by anjali1102 — APPROVED** [review]
Overall functionalities working 
clean UI
good to go

**Pull request merged**